doc/development.md

```diff
@@ -109,12 +109,17 @@ bundle and a custom catalog including the custom bundle.
 
 The `make bundle` target accepts the following variables:
 
-| **Makefile Variable** | **Description** | **Default value** |
-| --- | --- | --- |
-| `IMG` | Kuadrant operator image URL | `quay.io/kuadrant/kuadrant-operator:latest` |
-| `VERSION` | Bundle version | `0.0.0` |
-| `LIMITADOR_OPERATOR_BUNDLE_IMG` | Limitador operator bundle URL | `quay.io/kuadrant/limitador-operator-bundle:latest` |
-| `AUTHORINO_OPERATOR_BUNDLE_IMG` | Authorino operator bundle URL | `quay.io/kuadrant/authorino-operator-bundle:latest` |
+| **Makefile Variable**           | **Description**               | **Default value**                                   | **Notes**                                                        |
+|---------------------------------|-------------------------------|-----------------------------------------------------|------------------------------------------------------------------|
+| `IMG`                           | Kuadrant operator image URL   | `quay.io/kuadrant/kuadrant-operator:latest`         |                                                                  |
+| `VERSION`                       | Bundle version                | `0.0.0`                                             |                                                                  |
+| `TAG`                           | Kuadrant operator tag         | `latest`                                            | Needed if IMG is not provided, will build the operator image URL |
+| `LIMITADOR_OPERATOR_BUNDLE_IMG` | Limitador operator bundle URL | `quay.io/kuadrant/limitador-operator-bundle:latest` |                                                                  |
+| `AUTHORINO_OPERATOR_BUNDLE_IMG` | Authorino operator bundle URL | `quay.io/kuadrant/authorino-operator-bundle:latest` |                                                                  |
+| `RELATED_IMAGE_WASMSHIM`        | WASM shim image URL           | `oci://quay.io/kuadrant/wasm-shim:latest`           |                                                                  |
+| `LIMITADOR_OPERATOR_VERSION`    | Limitador operator version    | `latest`                                            | As an alternative to LIMITADOR_OPERATOR_BUNDLE_IMG               |
```

Comment:
This is confusing.

The `bundle` catalgo only uses `IMG`, `VERSION`, `TAG`, `LIMITADOR_OPERATOR_BUNDLE_IMG` , `AUTHORINO_OPERATOR_BUNDLE_IMG` and `RELATED_IMAGE_WASMSHIM`. 

`LIMITADOR_OPERATOR_VERSION` and `AUTHORINO_OPERATOR_VERSION` and `WASMSHIM_VERSION` are not  being used directly, only affecting the "default" value if not set. So I would not add them to the table. Just mention about them in the description of the default.